server/impl/src/main/java/com/walmartlabs/concord/server/events/github/GithubTriggerV2Processor.java

```diff
@@ -81,6 +102,26 @@ private void enrichEventConditions(Payload payload, TriggerEntry trigger, Map<St
         }
     }
 
+    private void disableRepo(RepositoryEntry repo, Payload payload) {
+        log.info("disable repo ['{}', '{}'] -> ref deleted", repo.getId(), payload.getBranch());
+        repoDao.disable(repo.getId());
```

Comment:
I think if we should wrap these two calls into a transaction.

Reply:
For combining in a transaction, is it better to duplicate the [trigger delete](https://github.com/walmartlabs/concord/blob/60b237267753d4dbfc7feedb21e6e6641a439b46/server/impl/src/main/java/com/walmartlabs/concord/server/org/triggers/TriggersDao.java#L79-L83) code in `RepositoryDao`? or inject `TriggersDao` into `RepositoryDao`?